plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/widgettype/validators/ContentListViewerWidgetValidator.java

```diff
@@ -119,7 +119,6 @@ protected void validateFilters(WidgetConfigurationRequest widget, BeanPropertyBi
                 && type.hasParameter(WIDGET_CONFIG_KEY_MAXELEMFORITEM)
                 && type.hasParameter(WIDGET_CONFIG_KEY_MAXELEMENTS)
                 && StringUtils.isNotEmpty((String) config.get(WIDGET_CONFIG_KEY_CONTENTTYPE))
-                && StringUtils.isEmpty((String) config.get(WIDGET_CONFIG_KEY_CATEGORIES))
```

Comment:
Sure about this?

Reply:
Yeah, it was throwing CastException with the new object format.
Didn't see a need to keep that kind of validation with a list.